1.0.0-beta4/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mono:3.12
+
+# Get build dependencies, download/build/install mono 4.1.0
+RUN apt-get update -qq \
+    && apt-get install -qqy git autoconf libtool automake build-essential mono-devel gettext unzip \
+    && git clone https://github.com/mono/mono.git \
+    && cd mono \
+    && git reset --hard 53dc56ee39a8e3b013231957aca4671b202c6410 \
+	&& ./autogen.sh --prefix="/usr/local" \
+    && make \
+    && make install \
+    && cd .. \
+    && rm mono -r
+
+# Install aspnet 1.0.0-beta4
+ENV DNX_FEED https://www.myget.org/F/aspnetrelease/api/v2
+ENV DNX_USER_HOME /opt/dnx
```

Comment:
I used the nightly Dockerfile as a reference and it doesn't set KRE_VERSION any more (https://github.com/aspnet/aspnet-docker). User can run "dnx --version" or "dnvm list" to get the current DNX version.